packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -112,10 +112,10 @@ export const addNpmDependency: AsyncAction<{
   async ({ actions, effects, state }, { name, version, isDev }) => {
     effects.analytics.track('Add NPM Dependency');
     state.currentModal = null;
-    let newVersion = version;
-
-    if (!newVersion) {
-      const dependency = await effects.api.getDependency(name);
+    let newVersion = version || 'latest';
+    const isTag = !newVersion.includes('.');
```

Comment:
Maybe we should rather check here whether the version is absolute or not. I have a function for this: https://github.com/codesandbox/codesandbox-client/blob/master/packages/common/src/utils/dependencies.ts#L35-L39, maybe we could use this for it?

Reply:
Fixed 👍 😄